src/molecules/Teaser/__test__/Teaser.test.js

```diff
@@ -17,6 +17,7 @@ const contributorsText = 'By Opie Schmuck and Opiette Schmuck';
 const description = 'This here is the description.';
 const headingLevel = 3;
 const TestImage = <Image image={image} />;
+const elementClass = 'testClass';
```

Comment:
This isn't too bad, but for clarity I'd recommend sticking things like default props into an object called "defaultProps" and grouping the other variables under some other wider group name.

For example, in Mimas I followed a pattern proposed by Dodds where I have a "defaultProps" function at the top of the page:

```
function defaultProps() {
  const srcSet =
    'https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/uncropped/35bd3b-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 400w,https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/uncropped/f5db37-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 600w,https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/uncropped/04a63f-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 1000w,https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/uncropped/72bc48-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 1400w,https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/uncropped/f20034-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 2000'
  const src =
    'https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/uncropped/f5db37-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg'
  const alt = 'Some nice lawn chairs'
  const fallbackSrc =
    'https://s3-us-west-2.amazonaws.com/s.cdpn.io/298/wolf_20131015_003_1400.jpg'
  const fallbackSrcSet =
    'https://s3-us-west-2.amazonaws.com/s.cdpn.io/298/wolf_20131015_003_1400.jpg 1400w, https://s3-us-west-2.amazonaws.com/s.cdpn.io/298/wolf_20131015_003_700.jpg 700w, https://s3-us-west-2.amazonaws.com/s.cdpn.io/298/wolf_20131015_003_400.jpg 400w'

  return {
    src,
    srcSet,
    alt,
    fallbackSrc,
    fallbackSrcSet
  }
}
```
Then, I run that setup function at the top of each test to get the props I need, and can even overwrite for special cases like `defaultProps.src = 'https://www.fillmurray.com/200/300'`.